README.md

```diff
@@ -1,27 +1,74 @@
+​
 
-## Making a plan
+## Make a plan of attack
 
-1. **Make a drawing of your app. Simple "wireframes"**
-1. **Look at the drawing and name the HTML elements you'll need to realize your vision**
-1. **Look at the drawing and imagine using the app. What _state_ do you need to track?**
-1. **For each HTML element ask: Why do I need this? (i.e., "we need div to display the results in")**
-1. **Once we know _why_ we need each element, think about how to implement the "Why" as a "How" (i.e., `resultsEl.textContent = newResults`)**
-1. **Find all the 'events' (user clicks, form submit, on load etc) in your app. Ask one by one, "What happens when" for each of these events. Does any state change? Does any DOM update?**
-1. **Think about how to validate each of your features according to a Definition of Done. (Hint: console.log usually helps here.)**
-1. **Consider what features _depend_ on what other features. Use this dependency logic to figure out what order to complete tasks.**
+1. Write HTML elements, label with IDs
+2. Create and export/import fetch-utils.js
+3. display list from fetched data
+4. Populate dropdown options from fetched data
+5. Apply search criteria
+6. Make page layout with CSS
+   ​
 
-Additional considerations:
+## HTML elements (stuff present upon page load)
 
--   Ask: which of your HTML elements need to be hard coded, and which need to be dynamically generated?
--   Consider your data model.
-    -   What kinds of objects (i.e., Dogs, Friends, Todos, etc) will you need?
-    -   What are the key/value pairs?
-    -   What arrays might you need?
-    -   What needs to live in a persistence layer?
--   Is there some state we need to initialize?
--   Ask: should any of this work be abstracted into functions? (i.e., is the work complicated? can it be reused?)
+-   img
+-   h1
+
+-   section
+-   h2
+-   form
+-   label
+-   select
+-   option
+-   button
+
+## State (everything you need to track internally using JS variables)
+
+-   let beanies = []
+-   let astroSigns = []
+
+## Events (anything that happens via JS when the user interacts with your site)
+
+-   window.addEventListener​('load', async () => {})
+-   async function findBeanies(astroSign) {}
+-   searchForm.addEventListener('submit', (e) => {})
+
+## Functions (to plan out how you'll segment things)
+
+​
+
+### Render Functions
+
+-   render-utils
+    -   export function renderBeanie(beanie) {}
+    -   export function renderAstroSign(astroSign) {}
+
+### Display functions
+
+-   function displayBeanies() {}
+-   function displayAstroOptions() {}
+
+### Fetch Functions (if applicable)
+
+-   in fetch-utils.js
+    -   export async function getBeanies(astroSign) {}
+    -   export async function getAstroSigns() {}
+
+### Other Functions
+
+​
+
+## Slices
+
+1. Populate HTML
+2. fetch-utils.js with supabase data (import supabase key and url)
+3. render-utils.js from supabase objects
+4. app.js to grab DOM elements
+5. styling
```

Comment:
I like that you plan out your code flow.  A very smart way to ensure everyone knows whats going on.